.github/workflows/build.yaml

```
@@ -316,7 +339,7 @@ jobs:
                     dependency-versions: "${{ matrix.dependencies }}"
 
             -   name: "Collect code coverage with pcov and phpunit/phpunit"
-                run: "vendor/bin/phpunit --coverage-clover=.build/logs/clover.xml"
+                run: "vendor/bin/phpunit tests/Unit/ --coverage-clover=.build/logs/clover.xml"
```

```suggestion
                run: "vendor/bin/phpunit --coverage-clover=.build/logs/clover.xml"
```

I think this should also include the functional tests. Don't you?